umibukela/templates/partner_detail.html

```diff
@@ -25,17 +25,17 @@
 
           <div class="row">
             <div class="col-md-6">
-              <h1>Folweni CRC</h1>
-              <h2>Folweni Community Resource Centre</h2>
+              <h1>{{ partner.short_name }}</h1>
+              <h2>{{ partner.full_name }}</h2>
               <a href="{% url 'site-result' site_name='folweni' result_id='4' %}" class="btn btn-primary btn-latest-results"><i class="fa fa-bar-chart"></i> View latest results</a>
             </div>
 
             <div class="col-md-6">
               <ul class="list-unstyled">
-                <li><i class="fa fa-fw fa-map-marker"></i> A1100 Folweni Township, Isipingo, 4110</li>
-                <li><i class="fa fa-fw fa-user"></i> <strong>Contact:</strong> Rose Dlamini</li>
-                <li><i class="fa fa-fw fa-phone"></i> <a href="tel:0319003454">031 900 3454</a></li>
-                <li><i class="fa fa-fw fa-envelope"></i> <a href="mailto:folwenicrc@telkomsa.net">folwenicrc@telkomsa.net</a></li>
+                <li><i class="fa fa-fw fa-map-marker"></i>{{ partner.physical_address }}</li>
+                <li><i class="fa fa-fw fa-user"></i> <strong>Contact:</strong> {{ partner.contact_person }}</li>
+                <li><i class="fa fa-fw fa-phone"></i> <a href="tel:{{ partner.telephone | slugify }}">{{ partner.telephone }}</a></li>
```

Comment:
hacky way to make it more url-ish - the example had spaces stripped but that wasn't trivial with a django filter - phones seem to strip hyphens but I was worried spaces might become %20. Only checked now -  https://tools.ietf.org/html/rfc3966#section-5.1.1 says spaces must not be used, but they have hyphens in the example. Do you think this solution is ok? I guess the "correct" way is to do output formatting in the view or even better, store it canonicalised.